content/pt-br/docs/reference/node/_index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Informações de referência do Node
```

Comment:
We should translate Node to "nó", as it is a common computing term in Portuguese.